tests/conftest.py

```diff
@@ -36,7 +36,7 @@
 
 def pytest_addoption(parser):
     parser.addoption(
-        "--server", action="store", metavar="SERVER:PORT", default="mockserverlocal:9543",
+        "--server", action="append", metavar="SERVER:PORT", default=[],
```

Comment:
@alanjcastonguay why isn't the default a list with a single item "mockserverlocal:9543"?

Reply:
Because the list couldn’t be replaced, only appended to. This way someone can specify to run against a live server and not the mock.

Reply:
Understood.  Thanks.